packages/grafana-ui/src/components/Combobox/Combobox.tsx

```diff
@@ -70,26 +75,61 @@ export const Combobox = ({ options, onChange, value, ...restProps }: ComboboxPro
       }
     },
   });
+
+  // the order of middleware is important!
+  const middleware = [
+    flip({
+      // see https://floating-ui.com/docs/flip#combining-with-shift
+      crossAxis: false,
+      boundary: document.body,
+      fallbackPlacements: ['top'],
+    }),
+  ];
+  const elements = { reference: inputRef.current, floating: floatingRef.current };
+  const { floatingStyles } = useFloating({
+    open: isOpen,
+    placement: 'bottom',
+    middleware,
+    elements,
+    whileElementsMounted: autoUpdate,
+  });
+
+  const hasMinHeight = isOpen && rowVirtualizer.getTotalSize() >= 400;
+
   return (
     <div>
-      <Input suffix={<Icon name={isOpen ? 'search' : 'angle-down'} />} {...restProps} {...getInputProps()} />
-      <div className={styles.dropdown} {...getMenuProps({ ref: listRef })}>
+      {/* @ts-expect-error */}
```

Comment:
Why are we expecting an error here?

Reply:
<img width="837" alt="Captura de pantalla 2024-07-17 a las 16 20 26" src="https://github.com/user-attachments/assets/a824829f-7b08-4152-ad84-9ccadd1f2b39">

Reply:
This seems like an incompatible type for onChange, wouldn't it be better to update it?